ksy_schema.json

```diff
@@ -241,7 +271,10 @@
             "pattern": "^(.*/)?[a-z][a-z0-9_]*$"
           }
         },
-        "encoding": { "type": "string" },
+        "encoding": {
+          "type": "string",
+          "description": "defines the default encoding used in all fields of type `str` or `strz` of the defining type and all it's children types"
```

Comment:
> 2\. The phrase _"used in **all** fields of type `str` or `strz`"_ is unnecessarily strong and it isn't accurate that this is the encoding used in _**all** fields with `str` type_, because the encoding can be overriden by the `encoding` key on the attribute level.

I think, actually that is accurate enough because of _default_ word: _all_ means that all fields have a default with the specified value. But your variant look better, so I'll use it

> You can also add a few examples of the most common encodings - for instance `utf-8`, `ascii`, etc.

I think that just examples are not enough, because this is a significant attribute for parsing, in difference from `tags`, for example. Therefore, the presence of examples is simply useless. The best I can do is put a link to https://github.com/kaitai-io/kaitai_struct/issues/116